src/Entity/Developer.php

```diff
@@ -65,6 +65,13 @@ class Developer extends EdgeEntityBase implements DeveloperInterface {
    */
   const APIGEE_EDGE_ERROR_CODE_DEVELOPER_DOES_NOT_EXISTS = 'developer.service.DeveloperDoesNotExist';
 
+  /**
+   * Developer email in the url doesnot match with the email in the request payload.
+   *
+   * @var string
+   */
+  const APIGEE_EDGE_ERROR_CODE_DEVELOPER_EMAIL_MISMATCH = 'rest.interceptor.name_mismatch';
```

Comment:
Maybe it should start with APIGEE_HYBRID or just APIGEE without "EDGE", since only the new Apigee products throws this exception code.

Reply:
Updated as `APIGEE_HYBRID`